content/04.network_applications.md

```diff
@@ -20,22 +20,22 @@ The output of this pipeline is an embedding space that clusters similar node typ
-This matrix ($A$) gets decomposed into three parts: a square matrix $Σ$ and a set of two small rectangular matrices $U$ and $V^{T}$.
-This values within $Σ$ are called singular values, which are akin to eigenvalues [@doi:10.1007/BF02288367].
+The input for this algorithm is an adjacency matrix ($A$), which is a square matrix where rows and columns represent nodes and each entry is a binary representation of the presence of an edge between two nodes. 
+$A$ is constructed based on the knowledge graph's structure itself and collapses all edges between two nodes into one unique entity.
+Following construction, $A$ is decomposed into the following three parts: a square matrix $Σ$ and a set of two small rectangular matrices $U$ and $V^{T}$.
+Values within $Σ$ are called singular values, which are akin to eigenvalues [@doi:10.1007/BF02288367].
 Each row in $U$ and each column in $V^{T}$ represents nodes within a low dimensional space [@doi:10.1007/BF02288367; @doi:10/bm8dnf].
-In practice $U$ is usually used to represent nodes in a knowledge graph, but $V^{T}$ can also be used [@doi:10.1007/BF02288367; @raw:Jiezhon2018].
-Typically, SVD appears in recommendation systems via collaborative filtering [@doi:10.1155/2009/421425]; however, this technique can also be used as a standalone baseline to compare to other approaches [@doi:10.1093/bioinformatics/btz718].
+In practice $U$ is usually used to represent nodes in a knowledge graph and can be used as input for machine learning classifiers to perform tasks such as link prediction or node classification [@doi:10.1093/bioinformatics/btz718]; however,$V^{T}$ can also be used [@doi:10.1007/BF02288367; @raw:Jiezhon2018].
+Typically, matrix factorization algorithms such as SVD appear in recommendation systems via collaborative filtering [@doi:10.1155/2009/421425]; however, this technique can also be used as a standalone baseline to compare to other relational learning approaches [@doi:10.1093/bioinformatics/btz718].
 
-Laplacian eigenmaps is a technique that assumes there is low dimensional structure in a high dimensional space [@doi:10.1162/089976603321780317].
-This algorithm preserves this structure while projecting data into a low dimensional space. 
-Typically, the first step of this algorithm is to construct a figurative knowledge graph where nodes represent datapoints and edges are constructed based on similarity of two datapoints; however, in this context, the knowledge graph has already been defined.
+Laplacian eigenmaps is a technique that assumes there is low dimensional structure in a high dimensional space and preserves this structure when projecting data into a low dimensional space [@doi:10.1162/089976603321780317].
+Typically, the first step of this technique is to construct a figurative graph where nodes represent datapoints and edges are constructed based on the similarity of two datapoints; however, in the context of knowledge graphs the graph has already been defined.
```

Comment:
By this do you mean that the knowledge graph is the graph that one needs? If so, maybe there's a way to make that clearer.